packages/testsweets/lib/src/constants/app_constants.dart

```diff
@@ -1 +1,3 @@
 const double WIDGET_DESCRIPTION_VISUAL_SIZE = 50;
+// should be a unique one to prevent conflict with user views name
+const String CAPTURED_VIEWS_KEY = '%CAPTURED_VIEWS_KEY%';
```

Comment:
Why are there `%` signs in this string?

Reply:
Cause I wanted to be as special as possible

Reply:
We can leave it like that for now then.

Reply:
We don't need it any more we can now search for the view name if it had an empty widget name